websites/B/bilibili/presence.ts

```diff
@@ -219,475 +232,28 @@ presence.on("UpdateData", async () => {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Documentary";
-      presenceData.state = "└ Science·Exploration·Nature";
-    } else if (
-      document.location.pathname.includes("/v/documentary/military/")
-    ) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Documentary";
-      presenceData.state = "└ Military";
-    } else if (document.location.pathname.includes("/v/documentary/travel/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Documentary";
-      presenceData.state = "└ Travel";
-    } else if (document.location.pathname.includes("/documentary/index/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Documentary";
-      presenceData.state = "└ Index";
-      //电影
-    } else if (document.location.pathname == "/movie/") {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-    } else if (document.location.pathname.includes("/v/movie/chinese/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-      presenceData.state = "└ Chinese";
-    } else if (document.location.pathname.includes("/v/movie/west/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-      presenceData.state = "└ European&American";
-    } else if (document.location.pathname.includes("/v/movie/japan/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-      presenceData.state = "└ Japanese";
-    } else if (document.location.pathname.includes("/v/movie/movie/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-      presenceData.state = "└ Other ";
-    } else if (document.location.pathname.includes("/movie/index/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing Movie";
-      presenceData.state = "└ Index";
-      //电视剧
-    } else if (document.location.pathname == "/tv/") {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing TV series";
-    } else if (document.location.pathname.includes("/v/tv/mainland/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing TV series";
-      presenceData.state = "└ China domestic";
-    } else if (document.location.pathname.includes("/v/tv/overseas/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing TV series";
-      presenceData.state = "└ China overseas";
-    } else if (document.location.pathname.includes("/tv/index/")) {
-      presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing TV series";
-      presenceData.state = "└ index";
```

Comment:
Why was this all removed?

Reply:
> Why was this all removed?

these things are very deep, and no one will go into these places. i am thinking of other more concise ways to implement them. some places that no one visits and take up too much code block. i think it is a bit too messy.

Reply:
If it works then there is no reason to remove it, even if close to noone goes there.

Reply:
> If it works then there is no reason to remove it, even if close to noone goes there.

i implemented it in a more concise way